src/main/fc/fc_core.c

```diff
@@ -275,10 +275,14 @@ void tryArm(void)
 
             if (!IS_RC_MODE_ACTIVE(BOXFLIPOVERAFTERCRASH)) {
                 flipOverAfterCrashMode = false;
-                pwmWriteDshotCommand(ALL_MOTORS, getMotorCount(), DSHOT_CMD_SPIN_DIRECTION_NORMAL);
+                if (!feature(FEATURE_3D)) {
```

Comment:
Wouldn't this better be added to https://github.com/betaflight/betaflight/pull/4729/files#diff-6d2ae6637303f59e8e3f33b77e79ba26L272 ?

Reply:
I still wanted it to get into the applyFlipOverAfterCrashModeToMotors() in mixer.c when enabled so you could do the flip with just 2 motors rotating.

Reply:
Ah, makes sense.